src/mnode/mnode-pastel.cpp

```diff
@@ -603,14 +603,77 @@ bool common_validation(const T& ticket, bool preReg, const std::string& strTnxId
     if (depth > 0)
         return true;
     
-    std::string err;
     if (!pastelTicket->IsValid(false, ++depth)) {
-      throw std::runtime_error(strprintf("The %s ticket with this txid [%s] is invalid - %s", prevTicket, strTnxId, err));
+      throw std::runtime_error(strprintf("The %s ticket with this txid [%s] is invalid", prevTicket, strTnxId));
     }
     
     return true;
 }
 
+void trade_copy_validation(const std::string& nftTnxId, const std::vector<unsigned char>& signature) {
```

Comment:
It would be nice to add comments in doxygen format (at least for some new functions) and describe parameters.

Reply:
added